ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.Data/Repositories/TaskRepository.cs

```diff
@@ -48,5 +59,35 @@ public Task BulkCreateAsync(List<Models.Task> tasks, CancellationToken cancellat
             cancellationToken.ThrowIfCancellationRequested();
             return GetCollection<Models.Task>().InsertManyAsync(tasks);
         }
+
+        public async Task<Dictionary<DayOfWeek, int>> GetHoursByDatesAsync(DateTime date, Guid employeeId, CancellationToken cancellationToken)
+        {
+            cancellationToken.ThrowIfCancellationRequested();
+            List<DateTime> dateList = new List<DateTime>();
+            List<string> testList = new List<string>();
+            var startDate = date;
+            var endDate = date.DeepCopy().AddDays(6);
+            for (int i = 0; i < 7; i++)
+            {
+                dateList.Add(date);
+                date = date.AddDays(1);
+            }
+
+            var filterBuilder = Builders<Models.Task>.Filter;
+            var employeeFilter = filterBuilder.Eq(t => t.EmployeeId, employeeId);
+            var dateFilter = filterBuilder.In(t => t.Date, dateList);
+            var testFilter = filterBuilder.Where(t => t.Date >= startDate && t.Date <= endDate);
+
+            var hours = await GetCollection<Models.Task>().Aggregate().Match(testFilter & employeeFilter).Group(
+                t => t.Date,
+                group => new
+                {
+                    date = group.Key,
```

Comment:
why date from lower case and Total from upper? should use upper for properties in .net